src/test/java/core/basesyntax/validation/impl/OutputValidatorImplTest.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.validation.impl;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class OutputValidatorImplTest {
+    private final String invalidFileForTest = "OutputValidatorinValidTest.csv";
+    private final String invalidQuantityFile = "resources/FileWithInvalidQuantity.csv";
+    private final OutputValidatorImpl outputValidator = new OutputValidatorImpl();
+
+    @Test
+    void fileValid_Ok() {
```

Comment:
which method are you testing here?) Let's follow the rules for naming test methods. Likewise in other cases